CONTRIBUTING.md

```diff
@@ -82,6 +82,17 @@ During testing, code coverage is measured. If the coverage is below defined thre
 Code coverage reports are generated in JSON and also in format compatible with _JUnit_. It is also possible to start `make coverage-report` to generate code coverage reports in form of interactive HTML pages. These pages are stored in `htmlcov` subdirectory. Just open index page from this subdirectory in your web browser.
 
 
+### Type hints checks
+
+It is possible to check if type hints added into the code are correct and whether assignments, function calls etc. use values of the right type. This check is invoked by following command:
+
+```
+make check-types
+```
+
+Please note that type hints check might be very slow.
```

Comment:
how long does it take to run currently?

Reply:
```
$ time make check-types
mypy ols/
Success: no issues found in 31 source files

real    0m31.912s
user    0m30.753s
sys     0m1.036s
```

Reply:
I did not add it into CI (well into `verify` target) because of the slow run. We can decide whether 30 seconds (and it will be more, let's say 1 minute) is ok or not.

Reply:
I think we should have it in our PR pre-submit testing for sure(30-60s is nothing in terms of PR test job time).  I have mixed feelings on whether it should be part of "make verify" by default given how long it takes.  For now i'd say make it optional when running locally but enabled in CI (either separate make target that we only invoke in the CI job, or based on an env var that we set in CI and individual developers can choose to set or not set).